ClickableUltrasound.cxx

```diff
@@ -7,14 +30,15 @@
 
 
 ClickableUltrasound::ClickableUltrasound(QWidget *parent) 
-: QLabel(parent) {
-	std::cout << "made clickableultrasond" << std::endl;
+: QLabel(parent) 
+{
 }
 
-void ClickableUltrasound::mousePressEvent(QMouseEvent * event) {
-	QPoint pos = event->pos();
-	//std::cout << pos.x() << ", " << pos.y() << std::endl;
-	if (mainWindow) {
-		mainWindow->OnUSClicked(pos);
-	}
+void ClickableUltrasound::mousePressEvent(QMouseEvent * event)
+{
+  QPoint pos = event->pos();
+  if (mainWindow)
+    {
```

Comment:
ITK just changed its style a few days ago to not indent brackets.  Just FYI.  Either way is fine for awhile.   ITK is publishing a clang style system to automatically format ITK code.